### DISCUSSION TITLE
PlanetScale exports?

### DISCUSSION THREAD

**Discussion by tessro** [planetscale/discussion#299] [Q&A] [answered]
Hi, we are evaluating switching from Aurora to PlanetScale.

While we are hoping to be happy customers for a long time, it's important for us to be confident we can migrate away should circumstances require it. I know binlogs aren't available. What export capabilities do you offer instead?

Thanks!
Paul

**Comment by samlambert** [comment, marked as answer]
Hi Paul,

We give you the ability to dump your database with `pscale database dump <database> <branch>`.

I hope this answers your question. If you would like some help migrating from Aurora please let us know.

**Comment by tessro** [comment]
Thanks! I have an open ticket about using the import feature but that's probably out of scope for this Q&A. :)

I've never tried to do a large scale MySQL dump, will that work well as we scale into millions of records, or are we kinda stuck once we reach a certain database size?

**Reply by samlambert** [reply]
I'll look into your import issue. 

Dump works to a pretty good size. Honestly if we let you down and you need help getting your data out we will help. We don't want you to have issues.

**Reply by tessro** [reply]
Thanks! Hard to imagine that scenario.

**Comment by rabinpoudyal** [comment]
dump seems to work but restore only seems to work on planetscale. Importing to other dbs is troublesome. Basically you get locked in
